docs/docsource/Operation-Architecture/Services/lifecycle_management/lifecycle_expiration_policies.rst

```diff
@@ -1,40 +1,42 @@
 Lifecycle Expiration Policies
 =============================
 
-Cloud users can apply lifecycle expiration rules (specified in Amazon’s
-`AWS S3 API <https://docs.aws.amazon.com/AmazonS3/latest/API/Welcome.html>`__)
+Cloud users can apply lifecycle expiration or transition rules (specified in 
+Amazon’s `AWS S3 API <https://docs.aws.amazon.com/AmazonS3/latest/API/Welcome.html>`__)
 to buckets managed through Zenko. These rules are triggered after a defined
-time has passed since the object’s creation.
+time has passed since the object’s creation. For the purposes of lifecycle 
+management, an object is "created" when it is touched.
 
-Zenko supports expiration of versioned or non-versioned objects, when a
-defined number of days has passed since those objects’ creation. This enables
-automatic deletion of older versions of versioned objects to reclaim storage
-space. Zenko also supports triggering expiration of current versions on a date
-from which a rule applies. This is currently feasible using the S3 API, but not
-with Orbit. Using Zenko from the command line or from Orbit, you can expire the
-current version of an object with a separate rule. For versioned buckets,
+Zenko supports expiration or transition of versioned or non-versioned objects,
+when a defined number of days has passed since those objects’ creation. This 
```

Comment:
~creation~ were last modified.

Reply:
Le mot juste! This fixes Dora's first concern as well.